servers/visual/visual_server_canvas.cpp

```diff
@@ -1463,7 +1471,16 @@ void VisualServerCanvas::canvas_item_attach_skeleton(RID p_item, RID p_skeleton)
 	}
 }
 
-void VisualServerCanvas::_canvas_item_skeleton_moved(RID p_item) {
```

Comment:
Note this function is renamed to `_canvas_item_invalidate_local_bound()`, as this behaviour is now shared between skeleton and multimesh.

Reply:
You've left `_canvas_item_skeleton_moved` declaration in the `.h` file, is this intended?

Reply:
Good catch, probably got reintroduced in the rebase. I'll remove it. :+1: